src/messages/MOSDPing.h

```diff
@@ -80,6 +99,10 @@ class MOSDPing : public Message {
     ::encode(op, payload);
     ::encode(peer_stat, payload);
     ::encode(stamp, payload);
+    if (min_message_size) {
```

Comment:
decoding is conditional on header.version, but encoding is conditional on whether min_emssage_size >= 0.  this should  be unconditional (encode empty bufferlist) or swtich the decode condition to p.end() (the former pbly?)